README.md

```diff
@@ -97,6 +97,13 @@ Follow this guide to setup your development machine.
 
     ```
     npm install
+    ``` 
+    Install bower(1.8.0) globally by running:
+    ```
+    npm install -g bower
+    ```
+    Now install the bower dependencies, run:
```

Comment:
Please change `Now install the bower dependencies, run:` -> `Now install the bower dependencies by running:`